src/lib/util/pair.c

```diff
@@ -53,6 +53,7 @@ void fr_pair_list_init(fr_pair_list_t *list)
 	 */
 	fr_pair_order_list_talloc_init(&list->order);
 
+	list->verified = true;
```

Comment:
these should be protected with `#ifdef WITH_VERIFY_PTR`, as with the other uses of the `list->verified` field.

Reply:
Will do.

Reply:
Done.